navigator_engine/common/graph_loader.py

```diff
@@ -1,5 +1,6 @@
 import navigator_engine.model as model

```

Comment:
This file is getting quite large and complex.  We should be careful to consider whether there is anyway we can refactor parts of the work into smaller functions. If it is not obvious or easy then don't worry about it, but definitely worth giving in a little thought.

Reply:
A low hanging fruit would be to make separate functions for creating different object in the model. e.g. _create_milestone, _create_conditional. Let's discuss how this fits in the priority list.